packages/rancher-gatekeeper/generated-changes/patch/templates/gatekeeper-manager-rolebinding-clusterrolebinding.yaml.patch

```diff
@@ -1,8 +0,0 @@
---- charts-original/templates/gatekeeper-manager-rolebinding-clusterrolebinding.yaml
-+++ charts/templates/gatekeeper-manager-rolebinding-clusterrolebinding.yaml
-@@ -15,4 +15,4 @@
- subjects:
- - kind: ServiceAccount
-   name: gatekeeper-admin
--  namespace: gatekeeper-system
-+  namespace: '{{ .Release.Namespace }}'
```

Comment:
Fixes that were removed seemed to have been introduced directly into the chart. LGTM